### ISSUE TITLE
bullet disappeared in ordered/unordered list if LaTeX code is present

### ISSUE THREAD

**Issue by AntSYau** [theme-next/hexo-theme-next#908]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
- $y=3x^2$
- i can't solve it

### Actual behavior 
    $y=3x^2$
- i can't solve it

### Steps to reproduce the behavior 
1. set math enable: true in next config
2. set mathjax: false in post header
3. bullets worked normally with equations not rendered
4. set mathjax: true in hexo post header and do hexo g
5. bullets are now disappeared in the lines with LaTeX equations

* Link to demo site with this bug: https://hblog.qhurc.com/2019/06/10/data-mining-revise/ (in Chinese)
* Original code for the first section:
````
## 第12章 大规模机器学习

###机器学习模型

应用机器学习算法的数据称为训练集。训练集由一系列$(x,y)$对组成，每个$(x,y)$称为一个**训练样例(training example)**，其中：
- $x$是一个**特征向量(feature vector)**
- $y$是一个**标签(label)**

ML过程的目标是寻找一个函数$y=f(x)$来预测对于每个$x$的最佳$y$值。$y$的常见重要类型：
- $y$是实数
- $y$是布尔值，或+1和-1 **$\Rightarrow$ 二类分类(binary classification)** 
- $y$是有穷集合中的元素 **$\Rightarrow$ 多类分类(multiclass classification)** 

####主要的机器学习算法类别：
- 决策树(decision tree)，类似于搜索树
- 感知机(preception)，对向量中各分量加权求和，若和大于给定阈值$\theta$则输出+1。适用于二类分类问题
- 神经网络(neural net)，感知机组成的网络。
- 基于实例的学习(instance-based learning)，使用整个训练集作为$f$。
````
### Node.js and NPM Information

```
v10.16.0
6.9.0
```

### Package dependencies Information

```
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "hexo": {
    "version": "3.8.0"
  },
  "dependencies": {
    "hexo": "^3.8.0",
    "hexo-deployer-git": "^1.0.0",
    "hexo-generator-archive": "^0.1.5",
    "hexo-generator-category": "^0.1.3",
    "hexo-generator-index": "^0.2.1",
    "hexo-generator-tag": "^0.2.0",
    "hexo-inject": "^1.0.0",
    "hexo-render-pug": "^2.1.4",
    "hexo-renderer-ejs": "^0.3.1",
    "hexo-renderer-kramed": "^0.1.4",
    "hexo-renderer-stylus": "^0.3.3",
    "hexo-server": "^0.3.3"
  }
}
```

#### Hexo version

```
v10.16.0
```

#### Hexo Configuration

```yml
theme: next
deploy:
  type: git
  repo: https://github.com/AntSYau/hexo-blog.git
  branch: master
```

### NexT Information

**NexT Version:**

- [x] Latest Master branch
- [ ] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [ ] All schemes
- [ ] Muse
- [x] Mist
- [ ] Pisces
- [ ] Gemini

#### NexT Configuration:

```yml
math:
  enable: true
```

### Other Information 
If your network speed is slow, you can acturally see the bullets disappear. They WERE rendered but somehow disappeared immediately after mathjax css is loaded.
![image](https://user-images.githubusercontent.com/10476297/59196864-67a08480-8bc2-11e9-8429-7ed6e52428e2.png)
![image](https://user-images.githubusercontent.com/10476297/59196885-7a1abe00-8bc2-11e9-922e-cc068bc746ce.png)

**Comment by stevenjoezhang** [comment]
See: https://github.com/theme-next/hexo-theme-next/issues/889, https://stackoverflow.com/questions/21769248/list-with-nested-overflow-x-hidden-hides-list-counter-point-why-is-this-a-b

**Comment by AntSYau** [comment]
if you don't know what happened even if you DID read issue #889, here's the direct solution:

1. go to %hexo_home%/themes/next/source/css/_custom
2. open custom.styl
3. append the following text to this file:
````
// Custom styles.

ul {
    list-style-position: inside
}

ol {
    list-style-position: inside
}
````
4. do
````bash
$ hexo clean
$ hexo g
$ echo don't forget to generate CNAME if you hosted your blog on github.io
$ hexo d
````

that theoretically should solve this problem. if you still can't figure out what you should do, consider changing the renderer to hexo-renderer-pandoc. that can solve this problem without changing configurations.

**Comment by AntSYau** [comment]
besides reference #745

**Comment by stevenjoezhang** [comment]
Fixed in #1004
